sqlalchemy-stubs/sql/sqltypes.pyi

```diff
@@ -252,8 +277,8 @@ class ARRAY(Indexable, Concatenable, TypeEngine[List[_T]]):
     @property
     def hashable(self) -> bool: ...  # type: ignore  # incompatible with supertype "TypeEngine"
     @property
-    def python_type(self) -> Type[list]: ...
-    def compare_values(self, x: _T, y: _T) -> bool: ...
+    def python_type(self) -> Type[List[_T]]: ...
+    def compare_values(self, x: Any, y: Any) -> bool: ...
```

Comment:
The function just does `x == y`, so it can take any values. I'll add a comment.

Reply:
Hm, sometimes mypy prohibits things that work at runtime. For example:
```python
def add(x: int, y: int) -> int:
    x + y
```
also works for strings before one adds types. Anyway, if you are sure it is _intended_ to be used with arbitrary objects, then `object` instead of `Any` would be better.

Reply:
OK